akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala

```diff
@@ -251,7 +252,7 @@ private[akka] object StreamLayout {
          * materialized value computation of its submodules.
          */
         Atomic(this),
-        Attributes.none)
+        DefaultAttributes.nested)
```

Comment:
Hmm, not sure I follow: Actor names are constructed from all Name attributes that are found in the list and only if none are found does it get called `unknown-operation`. Nesting modules should not add to the names generated, and not adding to the Attributes list should achieve that, no?